### DISCUSSION TITLE
Any documentation on app versioning?

### DISCUSSION THREAD

**Discussion by assassin316** [unoplatform/uno#14903] [Q&A] [answered]
Hi,

I was wondering if there is any documentation on versioning an app? Example, is there a way to specify a dynamic app version (which auto updates per build) in the **Directory.Build.props** file so that it can be read from Wasm, UWP, etc ?

**Comment by jeromelaban** [comment, marked as answer]
In uno, we're using [GitVersion](https://github.com/GitTools/GitVersion), but [Nerdbank.GitVersioning](https://github.com/dotnet/Nerdbank.GitVersioning) is also popular.

**Comment by assassin316** [comment]
Thanks, will plug in one of those 👍

**Reply by sasakrsmanovic** [reply]
Hello @assassin316 . Did one of those work?

**Reply by assassin316** [reply]
Hi @sasakrsmanovic , yes I used Nerdbank and was very easy to integrate 👍
